.github/workflows/dapr-bot-schedule.yml

```diff
@@ -1,61 +0,0 @@
-# ------------------------------------------------------------
```

Comment:
do not remove this file.

Reply:
fixed

Reply:
@mukundansundar pls review. I'm working on another feature branch and it would become hard merging when done... thanks!

Reply:
This file is still modified in the latest PR due to a probably unintended modification to the licensing header.